index.js

```diff
@@ -1 +1,4 @@
+console.log('jfkssl;adj')
+console.log('blablabla')
+console.log('blablabla')
```

Comment:
burayi boyle yapalim

```suggestion
console.log('hmmmm')
```